### DISCUSSION TITLE
PLY parser ouput in my workflow help

### DISCUSSION THREAD

**Discussion by sr1dar** [community/community#27276] [Actions]
Hello,

I use a Python script in my workflow to print some statements.
Today when I ran the workflow, I got a 20,000 line PLY output like this followed by my actual output (10 lines). 
A package that I use in this code is built with ply (lex & yacc in python)

This wasn't the case when I ran my workflow few weeks back. How do I fix this?
Thanks

![image](https://user-images.githubusercontent.com/100928638/181216815-23d686e2-6f20-4c5c-bc83-34aff9e9de9f.png)

**Comment by airtower-luna** [comment]
That's impossible to answer without knowing your code, especially the script called there. From your description my first guess is that *something* changed in those dependencies. Check if any versions were updated between the workflow runs, and if yes what changed between them. :smiley_cat: 

A quick fix would be to hide the output you don't want, but I wouldn't recommend that because it might hide errors or warnings in the future.
